src/plugins/three/QuantizedMeshPlugin.js

```diff
@@ -20,7 +20,7 @@ function isAvailable( layer, level, x, y ) {
 		available,
 	} = layer;
 
-	if ( level >= minzoom && level <= maxzoom && level < available.length ) {
+	if ( level >= ( minzoom ?? 0 ) && level <= ( maxzoom ?? Infinity ) && level < available.length ) {
```

Comment:
I didn't realize these fields could be undefined. Lets default them above instead of inlining these values:

```js
	const {
		minzoom = 0,
		maxzoom = Infinity,
		available,
	} = layer;
```

Reply:
Actually, I think `maxzoom` should not be present in the layer.json. As you did in your `maxLevel` getter, it should be `layer.available.length`. But let's keep it as you proposed just above. `maxLevel` and `layer.available.length` are not semantically exactly the same thing.

Reply:
`maxZoom` is present in the in the layer.json file but it doesn't necessarily seem to match `available.length`. There's no spec afaik for this file beyond the CesiumTerrainProvider file you linked below. The `maxZoom` field [is used in that file](https://github.com/CesiumGS/cesium/blob/46ed67c0bf601b52f3b1acdb7bb13d5decc5a181/Source/Core/CesiumTerrainProvider.js#L220-L221) but it's not clear how it differs from `available.length`, unfortunately. We may have to ask someone from Cesium at some point.